src/Component/Metadata/Index.php

```diff
@@ -30,6 +30,8 @@ public function __construct(
         string|callable|null $processor = null,
         ?bool $read = null,
         ?bool $write = null,
+        ?string $formType = null,
+        ?array $formOptions = null,
```

Comment:
WDYT about grouping some parts of these attributes into groups and defining nested attributes. Would it be possible to define some Form attribute and interface for it? So if anyone would like to change something, will it be possible? But TBH, the main reason behind this question is that once you would like to extend APIP attributes and you have 30 arguments long constructor it makes you a little bit sad